kinto/core/storage/postgresql/migrator.py

```diff
@@ -0,0 +1,95 @@
+"""
+A helper class to run migrations using a series of SQL files.
+"""
+
+import logging
+import os
+
+logger = logging.getLogger(__name__)
+
+
+class Migrator(object):
```

Comment:
No need to `(object)` in any py 3 code.

Reply:
Also, is `MigratorMixin` a better name to indicate to the code reader how you're supposed to use this?